### PR TITLE
[201811 Monit] Enable monitoring of SWSS daemons

### PR DESCRIPTION
**Pull request by zhenggen-xu**
[201811 Monit] Enable monitoring of SWSS daemons

Signed-off-by: Zhenggen Xu <zxu@linkedin.com>

This fix is only applicable to 201811 branch.

**- Why I did it**
Enable monitoring of SWSS daemons

**- How I did it**
See diff.

**- How to verify it**
**Before the fix**
```
sudo monit summary
Monit 5.20.0 uptime: 3d 12h 14m
┌─────────────────────────────────┬────────────────────────────┬───────────────┐
│ Service Name                    │ Status                     │ Type          │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ lnos-x1-a-csw04                 │ Running                    │ System        │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ rsyslog                         │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ telemetry                       │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ dialout_client                  │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ syncd                           │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ dsserve                         │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ snmpd                           │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ snmp_subagent                   │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ lldpd_monitor                   │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ lldp_syncd                      │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ lldpmgrd                        │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ redis_server                    │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ zebra                           │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ fpmsyncd                        │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ bgpd                            │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ root-overlay                    │ Accessible                 │ Filesystem    │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ var-log                         │ Accessible                 │ Filesystem    │
└─────────────────────────────────┴────────────────────────────┴───────────────┘
```

**After fix:**
```
 sudo monit summary
Monit 5.20.0 uptime: 3d 12h 13m
┌─────────────────────────────────┬────────────────────────────┬───────────────┐
│ Service Name                    │ Status                     │ Type          │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ lnos-x1-a-csw04                 │ Running                    │ System        │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ rsyslog                         │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ telemetry                       │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ dialout_client                  │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ syncd                           │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ dsserve                         │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ orchagent                       │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ portsyncd                       │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ neighsyncd                      │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ vrfmgrd                         │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ vlanmgrd                        │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ intfmgrd                        │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ portmgrd                        │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ buffermgrd                      │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ nbrmgrd                         │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ snmpd                           │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ snmp_subagent                   │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ lldpd_monitor                   │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ lldp_syncd                      │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ lldpmgrd                        │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ redis_server                    │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ zebra                           │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ fpmsyncd                        │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ bgpd                            │ Running                    │ Process       │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ root-overlay                    │ Accessible                 │ Filesystem    │
├─────────────────────────────────┼────────────────────────────┼───────────────┤
│ var-log                         │ Accessible                 │ Filesystem    │
└─────────────────────────────────┴────────────────────────────┴───────────────┘
```

**- Which release branch to backport (provide reason below if selected)**

- [X] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by yxieca** [comment]
@jleveque do you see issue with this change? It looks good to me.

File: dockers/docker-orchagent/base_image_files/monit_swss [hunk 1]
```diff
@@ -39,5 +38,3 @@ check process buffermgrd matching "/usr/bin/buffermgrd -l"
 check process nbrmgrd matching "/usr/bin/nbrmgrd"
     if does not exist for 5 times within 5 cycles then alert
 
-check process vxlanmgrd matching "/usr/bin/vxlanmgrd"
-    if does not exist for 5 times within 5 cycles then alert
```
Comment (jleveque) [inline dockers/docker-orchagent/base_image_files/monit_swss:43]: Why remove monitoring for vxlanmgrd in 201811?

Comment (zhenggen-xu) [reply dockers/docker-orchagent/base_image_files/monit_swss:43]: I don't think vxlanmgrd exists on 201811

Comment (jleveque) [reply dockers/docker-orchagent/base_image_files/monit_swss:43]: You're correct. I was looking at the wrong branch :)

**Comment by jleveque** [comment]
Retest vsimage please

**Pull request merged**